### PR TITLE
Feature/wallet

### PR DESCRIPTION
**Pull request by shirinmizan**

### CODE REVIEW

File: src/components/FundsBox/FundsBox.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <div className="fundsBox">
+        <div className="fundsBox__content">
+          <img
+            className="fundsBox_square"
+            src={blueSquare}
+            alt="blueSquareBox"
+          />
+          <h3 className="fundsBox__content__title">Your Funds:</h3>
+          <div className="fundsBox__content__amount">
+            {"\u00A3" + fundsAmount.toLocaleString("en-US")}
```
Comment (RenataY) [inline src/components/FundsBox/FundsBox.jsx:27]: All wallets should be in US currency as per the team agreement

File: src/App.jsx [hunk 2]
```diff
@@ -14,6 +14,10 @@ import BillingAddress from "./containers/BillingAddress/BillingAddress";
 import LandingMain from "./containers/LandingMain/LandingMain";
 import ContactListPage from "./components/ContactListPage/ContactListPage";
 import LiveRates from "./components/LiveRates/LiveRates";
+import Wallet from "./containers/Wallet/Wallet";
+
+// import Button from "./components/Button/Button"; // to be removed
```
Comment (RenataY) [inline src/App.jsx:19]: please remove commented buttons blocks

**Review by RenataY — CHANGES_REQUESTED** [review]
App doesn't start so can't really look at the changes.
userInfo is misspelled and can't be loaded (the file is UserInfo in the tree and userInfo in the Wallet.jsx.
Also there are other errors, looks like few things are now missing from App.jsx that should be there, please pull latest master to fix your branch.
 Line 108:73:  'searchTerm' is not defined   no-undef
  Line 108:98:  'handleInput' is not defined  no-undef
  Line 109:68:  'searchTerm' is not defined   no-undef
  Line 109:93:  'handleInput' is not defined  no-undef

Also please see other comments through Files changed

File: src/components/FundsBox/FundsBox.jsx (hunk 1 above)
Comment (shirinmizan) [reply src/components/FundsBox/FundsBox.jsx]: This is changed

**Comment by shirinmizan** [comment]
all the changes are fixed

**Review by RenataY — APPROVED** [review]
looks good

**Pull request merged**